src/app/vehicle-technical-record/vehicle-technical-record.component.spec.ts

```diff
@@ -28,4 +29,23 @@ describe('VehicleTechnicalRecordComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should get current vrm', () => {
+    component.vehicleTechRecord = mockVehicleTechnicalRecord();
+    expect(component.currentVrm).toEqual('KP01 ABC');
+  });
+
+  it('should other Vrms', () => {
```

Comment:
I think there's a word missing here

Reply:
It should get other VRMs at a guess? Could it be a more clear description ?